dora/core/client/fs/src/test/java/alluxio/client/file/cache/LocalCacheManagerTest.java

```diff
@@ -1089,6 +1094,25 @@ public void listPageIds() throws Exception {
             0).get(0));
   }
 
+  @Test
+  public void getDataFileChannel() throws Exception {
+    mCacheManager = createLocalCacheManager();
+    mCacheManager.put(PAGE_ID1, PAGE1);
+    CacheContext cacheContext = CacheContext.defaults();
+    Optional<DataFileChannel> dataFileChannel = mCacheManager.getDataFileChannel(PAGE_ID1,
+        0, PAGE1.length, cacheContext);
+    assertNotNull(dataFileChannel);
+    assertEquals(dataFileChannel.isPresent(), true);
+    assertEquals(dataFileChannel.get().getNettyOutput() instanceof DefaultFileRegion, true);
+    DefaultFileRegion defaultFileRegion =
+        (DefaultFileRegion) dataFileChannel.get().getNettyOutput();
+    ByteBuf buf = Unpooled.buffer(PAGE1.length);
+    NettyBufTargetBuffer targetBuffer = new NettyBufTargetBuffer(buf);
+    defaultFileRegion.transferTo(targetBuffer.byteChannel(), 0);
+    byte[] bytes = targetBuffer.byteArray();
+    assertArrayEquals(PAGE1, bytes);
```

Comment:
```suggestion
    ByteBuf buf = Unpooled.buffer(PAGE1.length);
    NettyBufTargetBuffer targetBuffer = new NettyBufTargetBuffer(buf);
    long bytesTransferred =
      defaultFileRegion.transferTo(targetBuffer.byteChannel(), 0);
    assertsEquals(PAGE1.length, bytesTransferred);
    byte[] bytes = new bytes[PAGE1.length];
    buf.readBytes(bytes);
    assertArrayEquals(PAGE1, bytes);
```
so that you don't have to call `ReadTargetBuffer.byteArray()`

Reply:
Updated.